test/sample_locations.json

```diff
@@ -12,16 +12,24 @@
     "location": "13/40.01264/-75.70446",
     "name": "downingtown_alt_trk_bus",
     "viewport": {
-      "width": 600,
-      "height": 600
+      "width": 400,
+      "height": 400
     }
   },
   {
     "location": "13/35.22439/-99.99495",
     "name": "historic_us66_oklahoma",
     "viewport": {
-      "width": 600,
-      "height": 600
+      "width": 400,
+      "height": 400
+    }
+  },
+  {
+    "location": "15/33.92208/-83.37654",
+    "name": "athens_ga_7_way_concurrency",
+    "viewport": {
+      "width": 400,
+      "height": 400
```

Comment:
I added this because it's a good sample location for multiple-concurrency shields and shield coloration.